.circleci/config.yml

```diff
@@ -2,29 +2,51 @@ version: 2.1
 setup: true
 orbs:
   prodsec: snyk/prodsec-orb@1
-  orb-tools: circleci/orb-tools@12.0.3
-  node: circleci/node@5.1.0
+  orb-tools: circleci/orb-tools@12.3.1
 
 defaults: &defaults
   docker:
-    - image: cimg/base:current
+    - image: cimg/node:lts
   working_directory: ~/repo
 
 commands:
   install_cci_cli:
     description: Install the latest CircleCI CLI
     steps:
+      - restore_cache:
```

Comment:
Storing the CCI CLI and `semantic-release` on a cache to stop downloading on every run.